### PR TITLE
RHV-16816 make production disk size prominent [enterprise-4.4][enterprise-4.5]

### PR DESCRIPTION
**Pull request by rolfedh**
Make the production environment disk size more prominent.
Make the VM template prerequisite more prominent.

### CODE REVIEW

**Review by peterclauterbach — APPROVED** [review]
LGTM

**Comment by lamek** [comment]
LGTM.

**Comment by jan-zmeskal** [comment]
Looks good from QE side

File: modules/installation-launching-installer.adoc [hunk 1]
```diff
@@ -101,6 +101,10 @@ ifndef::osp,rhv[* Configure an account with the cloud platform that hosts your c
 
 * Obtain the {product-title} installation program and the pull secret for your
 cluster.
+ifdef::rhv[]
+* xref:../../installing/installing_rhv/installing-rhv-creating-custom-vm.adoc#installing-rhv-creating-custom-vm[Create a custom virtual machine template on {rh-virtualization}]. Otherwise, the {product-title} cluster will fail.
```
Comment (vikram-redhat) [inline modules/installation-launching-installer.adoc:105]: There should be no xrefs in modules. @rolfedh can you remove this.

@lamek ^^

**Pull request merged**